src/components/AvailableQuestions/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Moment from 'react-moment'
+import { push } from 'connected-react-router'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+
+import Card from 'components/Card'
+
+import './availableQuestions.scss'
+
+const AvailableQuestions = props =>
+  props.data.map(question => (
+    <Card
+      className="cardQuestion flex justify-between"
+      onClick={() => props.push('/edit', { question })}
+      key={question.id}
+    >
+      <h6 className="cardQuestion__h6 ">{question.statement}</h6>
+      <h6>
+        <Moment format="DD/MM/YYYY">{question.created_at}</Moment>
```

Comment:
Acho que não precisamos de moment.js (que é tão pesado) para um caso tão simples. Acho que o seguinte deve resolver:
```
created_at.toLocaleDateString()
// ou (não sei o formato)
new Date(created_at).toLocaleDateString()
```
Se ainda assim quiserem ter uma biblioteca de datas, sugiro usar a date-fns.
Para outros casos:
[You Dont Need Momentjs](https://github.com/you-dont-need/You-Dont-Need-Momentjs)